ApplicationLibCode/ReservoirDataModel/RigEnsembleFractureStatisticsCalculator.cpp

```diff
@@ -63,6 +63,11 @@ RigHistogramData RigEnsembleFractureStatisticsCalculator::createStatisticsData(
 {
     std::vector<cvf::ref<RigStimPlanFractureDefinition>> defs = esf->readFractureDefinitions();
```

Comment:
Prefer fractureDefinitions as variable name, and consider use of auto

Reply:
Fixed.